### DISCUSSION TITLE
The lastPing field is 8 hours less than the actual time after upgrading skywalking【tag>9.2.0】

### DISCUSSION THREAD

**Discussion by frankliusd** [apache/skywalking#10525] [Q&A]
The lastPing field of the UI service query interface is 8 hours shorter than the actual time,Boss, what happened, ask for help，Thank you!

**Comment by frankliusd** [comment]
tag=9.2.0

**Comment by wu-sheng** [comment]
Most likely, your OAP or agent machine has wrong time zone setup.

**Comment by frankliusd** [comment]
Thank you for your reply. The time zone cannot be changed. Is there any other way?

**Reply by wu-sheng** [reply]
Time zone doesn't matter, but you need to match the time and timezone.

**Comment by frankliusd** [comment]
Thank you！
